src/Pulsar.Client/Internal/PartitionedProducerImpl.fs

```diff
@@ -197,6 +198,15 @@ type internal PartitionedProducerImpl<'T> private (producerConfig: ProducerConfi
                 |> Seq.max
                 |> channel.SetResult
 
+            | IsConnected channel ->
+
+                Log.Logger.LogDebug("{0} IsConnected", prefix)
+                producers
+                |> Seq.map (fun producer -> producer.IsConnected)
+                |> Seq.contains(false)
+                |> not
```

Comment:
I think you can replace it with
`Seq.forall _.IsConnected`

Reply:
this syntax does not work but I changed it to `forall`